platform/ios/resources/ja.lproj/Localizable.strings

```diff
@@ -43,6 +43,9 @@
 /* Action sheet title */
 "SDK_NAME" = "Mapbox iOS SDK";
 
+/* Developer-only SDK update notification; {latest version, in format x.x.x} */
+"SDK_UPDATE_AVAILABLE" = "現在Mapbox iOS SDK %1$@が入手できる：";
```

Comment:
I added this as a way to validate my localization implementation — should we take it out?